cmd/awscollector/Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN apk --update add ca-certificates
 #	Build Stage            #
 #			       #
 ################################
-FROM golang:1.18 AS prep-build
+FROM golang:1.20 AS prep-build
```

Comment:
Can we try to use that dependabot update action? It seems that it works for dockerfiles.

Reply:
We use [dbotconf](https://github.com/open-telemetry/opentelemetry-go-build-tools/tree/main/dbotconf) to autogenerate dependabot files. those should not be edited manually. I think we could look at dbotconf to see how easy it would be to add `Dockerfile` detection for this. It may be useful for others as well.

Reply:
It looks like it supports. https://github.com/open-telemetry/opentelemetry-go-build-tools/blob/main/dbotconf/internal/mods.go#L51
can you try?

Reply:
Done